go/host/enclave/guardian.go

```diff
@@ -655,7 +662,9 @@ func (g *Guardian) periodicBatchProduction() {
 			err := g.enclaveClient.CreateBatch(context.Background(), skipBatchIfEmpty)
 			if err != nil {
 				g.logger.Error("Unable to produce batch", log.ErrKey, err)
-				g.notifyEnclaveUnavailable("seq enclave could not produce batch")
+				g.markSequencerUnavailable("seq enclave could not produce batch")
+			} else {
+				g.markSequencerHealthy()
```

Comment:
when would this be called? If the enclave was not healthy, then how come it was asked to produce a batch?